app/components/navigation/navigation.module.scss

```diff
@@ -16,22 +16,32 @@
     background-color: colors.$blue;
     width: 100%;
     @media (min-width: 768px){
-        width: 10rem;
+        width: 15rem;
     }
     
 }
 
 .top-level-link {
-    padding: 0.5rem;
-    &::after {
-        content: '+';
-        position: absolute;
-        right: 0.5rem;
-    }
+    padding: 0.75rem;
+    line-height: 1rem;
+    position: relative;
+    text-wrap: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    padding-right: 1.75rem;
+    width: 100%;
+}
+.nav-dropdown::after {
+    content: '+';
+    display: block;
+    position: absolute;
+    top: 50%;
+    right: 0.5rem;
+    transform: translateY(-50%);
```

Comment:
noob: also not sure how you came up with these values especially `top` and `transform` but when I tried to read the docs my eyes went cross-eyed 😬 😵‍💫

Reply:
Ah, that's for vertical centering. CSS magic <3